get-started/generate-experiments.bash

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

But will it work if I use Zsh? Probably in general, just wondering if there's a more general shebang.

Very miinor

Reply:
My experience with Termux on Android led me to believe that finding `/usr/bin/env` in that exact location has the same probability as `/bin/bash`, so I started skipping this but you're right, that's a better practice. 

Fixed. 

Anyone who uses Zsh probably has Bash also installed. Bash is almost universal now and this particular script should run on both. The shell runs `/bin/bash generate.bash` (or `/usr/bin/env bash generate.bash`) when I call `./generate.bash`, so it doesn't matter if the user calls this from Zsh or Fish or any other shell.